framework/source/class/qx/ui/core/scroll/AbstractScrollArea.js

```diff
@@ -267,9 +257,10 @@ qx.Class.define("qx.ui.core.scroll.AbstractScrollArea",
     */
 
     /**
-     * Returns the boundaries of the pane.
+     * Returns the dimensions of the pane.
      *
-     * @return {Map} The pane boundaries.
+     * @return {{width: number, height: number}|null} The pane dimension in pixel. Contains
```

Comment:
This is a useful depiction when read in the code, but what's in the curly brackets is supposed to be a type. Does this render properly in the API viewer? What would typically be there is:
```
@return {Map|null} The pane dimensions in pixels. The Map contains two members: width and height, the respective number of pixels for that dimension.
```

I like what you've shown, but I don't think we should put it in one place but not globally. It would be a nice global change.

I'm open to accepting this change as is, if others feel it's appropriate.